src/libcadet/model/parts/MultiChannelConvectionDispersionOperator.cpp

```diff
@@ -830,6 +831,8 @@ bool MultiChannelConvectionDispersionOperator::configure(UnitOpIdx unitOpIdx, IP
 
 	setSparsityPattern();
 
+	_phaseTransitionModel = new LinearExchangeBase();
```

Comment:
this should be parameterized.
It seems like its not even used here, so please delete.